features/canvas-2d-context-alpha.yml

```diff
@@ -0,0 +1,6 @@
+name: Canvas 2D `alpha` parameter
```

Comment:
The `alpha` parameter is also used in WebGL, WebGL2, and the bitmaprender, but support for it is drastically different across the different contexts (ie confirmed no Safari support here, but there is support in Safari in the WebGL context), and it landed at very different times across the different contexts. I am going to separate them out for each context.

Reply:
Yes, I do think Devs will think of this separately, adding transparency to an image is one of the key reasons developers choose image formats (jpg vs png). And because of the limited support here it's not something we want to lump in together with the rest of the features.